cdk/bin/cdk.ts

```diff
@@ -6,8 +6,17 @@ const app = new App();
 new ManageFrontend(app, 'ManageFrontend-CODE', {
 	stack: 'support',
 	stage: 'CODE',
+	scaling: {
+		minimumInstances: 1,
+	},
+	domain: 'manage.code.theguardian.com.origin.membership.guardianapis.com',
```

Comment:
I think the TLD is missing `dev-`, hence this should be: `manage.code.dev-theguardian.com.origin.membership.guardianapis.com`.